server/routes/forms.ts

```diff
@@ -15,20 +16,19 @@ const {
   },
 } = config
 
-export default (formService: FormService) => (router) => {
+export default (formService: FormService, conditionsServiceFactory: ConditionsServiceFactory) => (router) => {
   router.get(
     '/curfewAddress/:bookingId',
     asyncMiddleware(async (req, res: Response<any, LicenceLocals>) => {
       const { bookingId } = req.params
-      const {
-        prisoner,
-        licence: { licence },
-        licenceStatus,
-      } = res.locals
+      const { prisoner, licence: licenceRecord, licenceStatus } = res.locals
 
+      const { licence } = licenceRecord
       const isBass = getIn(licenceStatus, ['decisions', 'bassReferralNeeded']) === true
       const isAp = getIn(licenceStatus, ['decisions', 'approvedPremisesRequired']) === true
 
+      const conditions = conditionsServiceFactory.forLicence(licenceRecord).getFullTextForApprovedConditions(licence)
```

Comment:
this has been pulled out of the service to the route as there didn't seem much need for the form service to interact with the condition service. 

It was easier to just deal with the full licence record here rather than to pass both through or to try and work out the version here with the condition factory and pass that through - to be read by the factory again at the lower level.

Also made adjusting the tests much more straightforward